core/src/test/java/io/jenkins/pluginhealth/scoring/probes/DependabotPullRequestProbeTest.java

```diff
@@ -125,23 +134,30 @@ void shouldAccessGitHubAPIWhenDependabotActivatedWithOpenedPR() throws IOExcepti
         final GHLabel dependenciesLabel = mock(GHLabel.class);
         when(dependenciesLabel.getName()).thenReturn("dependencies");
 
+        final Date thirtyDaysAgo = getThirtyDaysAgoAsDate();
+        final Date ninetyOneDaysAgo = getNinetyOneDaysAgoAsDate();
+
         final GHPullRequest pr_1 = mock(GHPullRequest.class);
         when(pr_1.getLabels()).thenReturn(List.of(dependenciesLabel));
+        doReturn(ninetyOneDaysAgo).when(pr_1).getCreatedAt();
         final GHPullRequest pr_2 = mock(GHPullRequest.class);
+        doReturn(thirtyDaysAgo).when(pr_2).getCreatedAt();
```

Comment:
To be needed, we need to have the label `dependencies` on the pull request.
```suggestion
        doReturn(List.of(dependenciesLabel)).when(pr_2).getLabels();
        doReturn(thirtyDaysAgo).when(pr_2).getCreatedAt();
```